### PR TITLE
Set up branch

### PR DESCRIPTION
**Pull request by Yors-git**
In this branch we did the following:
- Set up the main project file structure
- Add the bin and lib folders
- Add main.rb and make it executable
- Set up code linters

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team! 🖥️ 👀 ✨ 

- Your project is almost perfect.

## However, some changes are still required... 🧰 🔨 🔧 

- Please change the name of your `main` file to `main.rb`, as specified by the project requirements.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
+puts 'Hello world!'
```
Comment (moinkhanif) [inline bin/main.rb:2]: [OPTIONAL] Please make this file executable as suggested. It's current permissions are: `-rw-r--r--`

**Review by moinkhanif — APPROVED** [review]
##  Status: Approved! 🥳  

Hey team, great work on your project ✨. You have completed the changes requested by the previous TSE's. However, there's still one small step as mentioned in the below comment. I'll be marking it as optional for now since it was not mentioned before. Please complete it soon. **I am approving** your project. 🎉

You may now merge the branch. Hope you enjoy upcoming projects and understand the concepts better! 😄

Happy coding! 🐱‍🏍

File: bin/main.rb (hunk 1 above)
Comment (Yors-git) [reply bin/main.rb:2]: We changed the permissions

**Pull request merged**